src/main/java/frc/robot/subsystems/Rotors.java

```diff
@@ -24,50 +24,49 @@ public class Rotors extends SubsystemBase {
   final TalonFX leftflywheelMotor = new TalonFX(mapRotors.LEFT_FLYWHEEL_CAN);
   final TalonSRX transferMotor = new TalonSRX(mapRotors.TRANSFER_CAN);
   final TalonFX climber = new TalonFX(RobotMap.mapRotors.CLIMBER_CAN);
-  
 
   public Rotors() {
     intakeMotor.getConfigurator().apply(ConstRotors.INTAKE_CONFIGURATION);
     rightHopperMotor.getConfigurator().apply(ConstRotors.HOPPER_RIGHT_CONFIGURATION);
     leftHopperMotor.getConfigurator().apply(ConstRotors.HOPPER_LEFT_CONFIGURATION);
     rightflywheelMotor.getConfigurator().apply(ConstRotors.FLYWHEEL_CONFIGURATION);
     leftflywheelMotor.getConfigurator().apply(ConstRotors.FLYWHEEL_CONFIGURATION);
-  transferMotor.configFactoryDefault();
-   climber.getConfigurator().apply(ConstRotors.CLIMBER_CONFIGURATION);
+    transferMotor.configFactoryDefault();
+    climber.getConfigurator().apply(ConstRotors.CLIMBER_CONFIGURATION);
   }
 
   final MotionMagicVelocityVoltage flywheelVelocityRequest = new MotionMagicVelocityVoltage(0);
 
-
-  public void setIntakeMotorSpeed(double speed) {
-    intakeMotor.set(speed);
+  public void setIntakeMotorPower(double power) {
+    intakeMotor.set(power);
   }
 
-  public void setHopperRightMotorSpeed(double speed) {
-    rightHopperMotor.set(speed);
+  public void setHopperRightMotorPower(double power) {
+    rightHopperMotor.set(power);
   }
 
-  public void setHopperLeftMotorSpeed(double speed){
-    leftHopperMotor.set(speed);
+  public void setHopperLeftMotorPower(double power) {
+    leftHopperMotor.set(power);
 
   }
-  public void setFlywheelMotorSpeed(AngularVelocity speed) {
+
+  public void setFlywheelMotorPower(AngularVelocity speed) {
```

Comment:
Can we name this one to velocity? Since it's not setting power it's targeting a velocity. We could even name it targetFlywheelMotorVelocity since PID is only ever trying to get close. I'll let you decide what makes more sense, feel free to ask other team members their opinion :)
```suggestion
  public void setFlywheelMotorVelocity(AngularVelocity speed) {
```